### ISSUE TITLE
What with this library?

### ISSUE THREAD

**Issue by tyt34** [cashbit/brain-predict#3]
If I try install this library, I see erorr:

> r@r-N61Vg ~/pr/ $ npm install brain-predict
> npm WARN checkPermissions Missing write access to /home/r/node_modules/brain-predict
> npm WARN enoent ENOENT: no such file or directory, open '/home/r/package.json'
> npm WARN sinon-chai@2.14.0 requires a peer of sinon@^1.4.0 || ^2.1.0 || ^3.0.0 || ^4.0.0 but none is installed. You must install peer dependencies yourself.
> npm WARN r No description
> npm WARN r No repository field.
> npm WARN r No README data
> npm WARN r No license field.
> 
> npm ERR! path /home/r/node_modules/brain-predict
> npm ERR! code EACCES
> npm ERR! errno -13
> npm ERR! syscall access
> npm ERR! Error: EACCES: permission denied, access '/home/r/node_modules/brain-predict'
> npm ERR! { Error: EACCES: permission denied, access '/home/r/node_modules/brain-predict'
> npm ERR! stack: 'Error: EACCES: permission denied, access '/home/r/node_modules/brain-predict'',
> npm ERR! errno: -13,
> npm ERR! code: 'EACCES',
> npm ERR! syscall: 'access',
> npm ERR! path: '/home/r/node_modules/brain-predict' }
> npm ERR!
> npm ERR! The operation was rejected by your operating system.
> npm ERR! It is likely you do not have the permissions to access this file as the current user
> npm ERR!
> npm ERR! If you believe this might be a permissions issue, please double-check the
> npm ERR! permissions of the file and its containing directories, or try running
> npm ERR! the command again as root/Administrator (though this is not recommended).
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR! /home/r/.npm/_logs/2018-06-25T17_14_59_122Z-debug.log
> 

What I do incorrect?

**Comment by cashbit** [comment]
SUDO

**Comment by tyt34** [comment]
if use sudo

> r@r-N61Vg ~/pr/node.JS/20_NN_synaptic $ sudo npm install brain-predict
> npm WARN deprecated brain@1.0.0: Package no longer supported. Contact support@npmjs.com for more info.
> npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
> npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
> 
> + brain-predict@0.0.8
> added 2 packages from 1 contributor and audited 1860 packages in 11.49s
> found 6 vulnerabilities (2 low, 3 high, 1 critical)
>   run `npm audit fix` to fix them, or `npm audit` for details
> 

After I start example, I see:

> 
> nodejs 2_25-06-18.js 
> module.js:529
>     throw err;
>     ^
> 
> Error: Cannot find module '../brain-predict'
>

**Comment by cashbit** [comment]
Right.... brain as a new life, updated package.json and package itself right now, see 
here: https://github.com/harthur/brain [DISCONTINUED]
and here: https://github.com/BrainJS/brain.js [new life]

**Comment by tyt34** [comment]
Even if I use 

> var predict = require("./brain-0.6.3.js") ;

I have erorr with:

> 
> var prediction = predict.predict(config) ;
>                          ^
> TypeError: predict.predict is not a function

**Comment by elan69** [comment]
try this `npm install -g brain.js`
